client/tracing/src/lib.rs

```diff
@@ -181,14 +261,17 @@ impl ProfilingSubscriber {
 }
 
 fn print_log(span_datum: SpanDatum) {
-	let message = format!(
+	let mut message = format!(
 		"Tracing: {} {}: {}, line: {}, time: {} ns",
 		span_datum.level,
 		span_datum.target,
 		span_datum.name,
 		span_datum.line,
-		span_datum.overall_time.as_nanos()
+		span_datum.overall_time.as_nanos(),
 	);
+	if !span_datum.values.0.is_empty() {
+		message = format!("{}, values: {}", message, span_datum.values);
```

Comment:
Is this the hot path that is being called by the macro that wants to log something?

If yes, could we prevent to allocate here?